source/tests/test_nvnmd_entrypoints.py

```diff
@@ -32,22 +33,30 @@ def test_freeze(self):
             "layer_0_type_0/bias",
             "final_layer_type_0/matrix",
             "final_layer_type_0/bias",
-        )
+        ]
+        namelist2 = [
+            "train_attr/min_nbor_dist"
+        ]
+        namelist = namelist1 + namelist2
+        # crete variable as namelist
         tvlist = []
-        save_path = str(tests_path / os.path.join("nvnmd", "weight.npy"))
+        save_path = str(tests_path / os.path.join("nvnmd/out", "weight.npy"))
         vinit = tf.random_normal_initializer(stddev=1.0, seed=0)
         for sname in namelist:
             scope, name = sname.split('/')[0:2]
             with tf.variable_scope(scope, reuse=False):
-                tv = tf.get_variable(name, [1], tf.float32, vinit)
-                tvlist.append(tv)
-        #
+                if sname in namelist1:
+                    tv = tf.get_variable(name, [1], tf.float32, vinit)
+                    tvlist.append(tv)
+                elif sname in namelist2:
+                    ts = tf.constant(2.0,
+                        name = name,
+                        dtype = tf.float64)
+        # test
```

Comment:
It's not clear to me what `tv` and `ts` are used for.

Reply:
To check the frozen model function, we create a network with the specified variables (tv) and constant tensor (ts).

Reply:
Got it.